spk/domoticz/src/dsm-control.sh

```diff
@@ -16,6 +16,14 @@ PORT="8084"
 
 start_daemon ()
 {
+    insmod /lib/modules/usbserial.ko
+    insmod /lib/modules/ftdi_sio.ko
+    insmod ${INSTALL_DIR}/modules/cp210x.ko
```

Comment:
Not sure, but shouldn't there be a corresponding `rmmod` when stopping/removing the package?